cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -238,8 +240,49 @@ func createConfigAPIEntry(c config.Config) (persistence.DeleteEntry, error) {
 		return persistence.DeleteEntry{}, fmt.Errorf("'name' parameter was empty")
 	}
 
+	var scopeValue string
+	if api.NewAPIs()[c.Coordinate.Type].HasParent() {
+		scopeParam, ok := c.Parameters[config.ScopeParameter]
```

Comment:
maybe i am overlooking something but i think it needs these 10+ steps to follow a reference and get a value of a parameter from it......